plugins/woocommerce/src/Admin/RemoteInboxNotifications/ComparisonOperation.php

```diff
@@ -64,6 +64,8 @@ public static function compare( $left_operand, $right_operand, $operation ) {
 					return strpos( $left_operand, $right_operand ) === false;
 				}
 				break;
+			case 'range':
+				return $left_operand >= $right_operand[0] && $left_operand <= $right_operand[1];
```

Comment:
Should we check if `$right_operand` is an array and has at least 2 elements like what we have in other cases?

Reply:
@chihsuan Good suggestion! I'll make the change.

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/45201/commits/d48f340724e180725548a5dfcb0d80a835de99a2